test/TEST-55-OOMD/test.sh

```diff
@@ -6,6 +6,23 @@ TEST_DESCRIPTION="systemd-oomd Memory Pressure Test"
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
 
+test_create_image() {
```

Comment:
can this work with `test_append_files()` ?

Reply:
It definitely can, thanks! Completely forgot about this feature.